modules/packages/UnorderedCopy.chpl

```diff
@@ -22,10 +22,16 @@
      This module represents work in progress. The API is unstable and likely to
      change over time.
 
-   This module provides an unordered version of copy/assign for ``numeric``
-   types. The results from this function are not visible until task or forall
-   termination or an explicit :proc:`unorderedCopyTaskFence()`, but it can
-   provide a significant speedup for bulk assignment operations that do not
+   This module provides an unordered version of copy/assign for trivially
+   copyable types types. Trivially copyable types are types that can be
+   duplicated by copying bits. They include ``numeric`` and ``bool`` types as
+   well as ``numeric``/``bool`` tuples and ``numeric``/``bool`` records
+   provided the record has no user-defined initializers, deinitializers, or
+   assignment overloads.
```

Comment:
How about

This module provides an unordered version of copy/assign for trivially copyable types types. Trivially copyable types are types require no special behavior to be copied and merely copying their representation is sufficient.  They include ``numeric`` and ``bool`` types as tuples or records consisting only of ``numeric``/``bool``. Records with no user-defined copy-initializers, deinitializers, or assignment overloads.

Note that just having an initializer shouldn't impact POD-ness; it's the copy initializer/init= we care about.